picodom.d.ts

```diff
@@ -0,0 +1,26 @@
+export interface VNode<D> {
+  tag: string
+  data?: D
+  children: VElement<{}>[]
```

Comment:
Children is always an array. If you pass a string to h(), it will return Array<string>.